index.html

```diff
@@ -237,6 +238,12 @@ <h3>Build Industry Related UX Portfolio</h3>
                 </div>
                 <div>Drop down list + icons + time + link or icon</div>
             </div>
+
+        </div>
+        <div class="row">
+            <div class="col-12">
+                <button><a href="#">Download Syllabus</a></button>
+            </div>
```

Comment:
As ji perkela i apacia nou 241 iki 246 nes jis prasideda nuo mobilios versijos is apacios.
O po to as planuoju ji su css position: absolute; pastatyti i vieta ne su col12 neiseina.

Reply:
Ai, taigi viršuje prie pull request rašei.
O šitem klasių nereikia? Vėliau? 
Drop down list + icons + time + link or icon

Reply:
As nezinau gal kita issues kay css bus?